content/guides/10-minute-tutorial.md

```diff
@@ -792,12 +793,281 @@ Feature: Is it Friday yet?
 
 Congratulations! You've got your first green Cucumber scenario.
 
+# Evaluate a scenario
+Now that we've forced a scenario to pass, let's write one that passes or fails based on logic. Let's update our statement to evaluate whether or not `today` is equal to `"Friday"`.
```

Comment:
Instead of adding the logic here, I'd prefer to add a test that tests for Friday which fails (currently) and *only then* adding the logic. (as this is the way we would actually do it using TDD/BDD). So first add the scenario from "Multiple scenarios" below, then add the logic.

Reply:
Sure, so something like `#See a scenario fail` before `#Evaluate a scenario`. I'll rearrange that.

Reply:
Thanks! That would be great. :)